server/api/organisation/organisation.controller.js

```diff
@@ -45,8 +45,13 @@ const putOrganisation = async (req, res) => {
 
   await Organisation.findByIdAndUpdate(req.params._id, { $set: req.body })
   res.json(await Organisation.findById(req.params._id).exec())
+  // Domain string validation
```

Comment:
Upon reading this I realise the way I originally did this (in `person.controller.js`) could be done better, by simply calling the validation function itself as opposed to using the regex.

Make use of the validation method:
```const orgValidation = require('./organisation.validation')
...
if (Object.keys(req.body).includes('domainName') && !orgValidation.domainName(req.body.domainName)) {
  return res.status(400).send('The \'domainName\' field does not match the validation rule')
}
```

Reply:
@bernesemountaindog  So i added the new line of code that you gave to the org controller. 
I'm going to make one validation file so that it can cover the person and organisations